vendor/models/subscription.py

```diff
@@ -97,4 +97,16 @@ def get_last_payment_date(self):
 
         return payment.get_receipt().start_date
         
+    def is_on_trial(self):
+        if not self.receipts.count():
+            return False
+
+        receipt = self.receipts.order_by('start_date').first()
```

Comment:
should this function filter the receipts by deleted=False as well?

Reply:
duh, thanks for catching that